tests/components/sonos/conftest.py

```diff
@@ -107,13 +109,23 @@ def __init__(self, return_value: dict[str, str], ip_address="192.168.42.2") -> N
 class SonosMockEvent:
     """Mock a sonos Event used in callbacks."""
 
-    def __init__(self, soco, service, variables) -> None:
+    def __init__(
+        self,
+        soco: MockSoCo,
+        service: SonosMockService,
+        variables: dict[str, str],
+        uui_ds: str | None = None,
```

Comment:
what is an uui_d?

Reply:
uui_ds is an optional XML element that may be present in a Sonos zone topology event. Sonos chose the name.

Reply:
An option is to update it to use the full name that Sonos sends in the event which is 

zone_player_uui_ds_in_group

That's probably the best so that searching for this attribute will also return this mock function parameter.